src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -51,48 +52,32 @@ def test_container_basics(self):
         Test basic container create/destroy/open/close/query.  Nothing fancy
         just making sure they work at a rudimentary level
 
-        :avocado: tags=container,containercreate,containerdestroy,basecont
+        :avocado: tags=pr,container,containercreate,containerdestroy,basecont
         """
-
-        pool = None
-        hostlist = None
-
         try:
-            hostlist = self.params.get("test_machines", '/run/hosts/*')
-            hostfile = write_host_file.write_host_file(hostlist,
-                                                       self.workdir)
+            self.hostlist_servers = self.params.get("test_machines", '/run/hosts/*')
```

Comment:
Lines 58-63 could be removed if you changed the inheritance to `TestWithServers`.  You could also remove lines 46-48 and 105-107 with this change.